core/src/main/java/com/linecorp/armeria/common/CommonPools.java

```diff
@@ -36,6 +38,13 @@ public final class CommonPools {
     private static final EventLoopGroup WORKER_GROUP =
             EventLoopGroups.newEventLoopGroup(Flags.numCommonWorkers(), "armeria-common-worker", true);
 
+    static {
+        // Bind EventLoopMetrics for the common worker group.
+        MoreMetrics
+                .eventLoopMetrics(new MeterIdPrefix("armeria.netty.common"), WORKER_GROUP)
```

Comment:
How about using `armeria.netty` and changing `MoreMetrics.eventLoopMetrics()` to accept the name of the event loop group? Here, we could pass `common` as the name. It would also align with Micrometer's other binders such as `ExecutorServiceMetrics`.

Reply:
Sure.